cuenca/resources/base.py

```diff
@@ -77,6 +79,32 @@ def _update(
         return cls._from_dict(resp)
 
 
+@dataclass
+class Downloadable(Resource):
+    @classmethod
+    def download(
+        cls,
+        instance,
+        *,
+        session: Session = global_session,
+        file_format: FileFormat,
+    ) -> BytesIO:
+        resp = session.download(
+            f'/{cls._resource}/{instance.id}',
+            headers=dict(Accept=file_format.value),
+            stream=True,
+        )
+        return BytesIO(resp)  # type: ignore
```

Comment:
Why are you ignoring the type? This should only happen if there's a good reason to do so

Reply:
I modify!